jerry-core/jcontext/jcontext.c

```diff
@@ -19,6 +19,50 @@
  * @{
  */
 
+#ifdef JERRY_ENABLE_EXTERNAL_CONTEXT
+
+/**
+ * Get the current context from the current instance.
+ *
+ * @return the pointer to the context
+ */
+jerry_context_t *
+jerry_context_get_current_context (void)
```

Comment:
I think in most embedded cases, binary/memory size is more important. Usually the heavy computation is handled by native binding, and JS is more like a task dispatcher, so its performance affects the global performance a little.